Command/GenerateAdminCommand.php

```diff
@@ -236,6 +249,39 @@ protected function interact(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @param string $bundleName
+     *
+     * @return string
+     */
+    private function determineConfigDir($bundleName)
+    {
+        $bundleConfigDir = $this->getBundle($bundleName)->getPath().'/Resources/config/';
+
+        if (!is_dir($bundleConfigDir)) {
+            return $this->getKernel()->getRootDir() . '/config';
+        }
+
+        $configFiles = $this->findServicesConfigFiles($bundleConfigDir);
+        if (count($configFiles) !== 0) {
+            return $bundleConfigDir;
+        }
+
+        return $this->getKernel()->getRootDir() . '/config';
+    }
+
+    /**
+     * @param string $configDir
+     *
+     * @return Finder|SplFileInfo[]
+     */
+    private function findServicesConfigFiles($configDir)
+    {
+        $finder = new Finder();
```

Comment:
Dependencies should be injected (otherwise you can't mock them in tests, among other things).